VigneshApp/src/styles.css

```diff
@@ -0,0 +1,4 @@
+/* You can add global styles to this file, and also import other style files */
```

Comment:
Why you keep css file

Reply:
css file created when compiling scss file, since i need to include tailwind in css